airflow/serialization/serialized_objects.py

```diff
@@ -789,6 +825,27 @@ def from_dict(cls, serialized_obj: dict) -> 'SerializedDAG':
             raise ValueError(f"Unsure how to deserialize version {ver!r}")
         return cls.deserialize_dag(serialized_obj['dag'])
 
+    @classmethod
+    def _serialize_dag_params(cls, dag_params: Dict[str, Param]):
+        """ """
+        serialized_params = {}
+        for k, v in dag_params.items():
+            serialized_params[k] = v.__dict__
+            serialized_params[k].update({'__type': f'{v.__class__.__module__}.{v.__class__.__name__}'})
```

Comment:
Isn't this the same as `v.dump()`?

Reply:
Additionally: everywhere ls `__type` is one of an enum, this should be `__class` or similar.